src/dev/build/tasks/os_packages/docker_generator/run.js

```diff
@@ -29,15 +29,26 @@ const linkAsync = promisify(link);
 const unlinkAsync = promisify(unlink);
 const chmodAsync = promisify(chmod);
 
-export async function runDockerGenerator(config, log, build) {
+export async function runDockerGenerator(config, log, build, ubi = false) {
+  // UBI var config
+  const baseOSImage = ubi ? 'registry.access.redhat.com/ubi7/ubi-minimal:7.7' : 'centos:7';
```

Comment:
do we have any plans to base this off our own, upstream base image?  not too concerned about it but thought we had one for some reason

Reply:
I think the current plan, across the stack, is to keep shipping centos as the base for oss and default and then an extra `ubi` based docker image just for the default distribution 😃

Reply:
👍